docs/projects/t1-2022/thoth-tech-leadership/weekly-announcements/week8.md

```diff
@@ -0,0 +1,19 @@
+#Week 8 Announcement
```

Comment:
**issue:** We will need to have a space between `#` and the header text, or else it won't render correctly.

```suggestion
# Week 8 Announcement
```